sippy-ng/src/component_readiness/CompReadyTestReport.js

```diff
@@ -157,6 +158,19 @@ export default function CompReadyTestReport(props) {
     setVersions(tmpRelease)
   }, [releases])
 
+  // this backhand way of recording the query dates keeps their display
+  // from re-rendering to match the controls until the controls update the report
+  const datesEnv = useContext(CompReadyVarsContext)
+  useEffect(() => {
+    let tmpDates = {
+      baseStartTime: datesEnv.baseStartTime,
+      baseEndTime: datesEnv.baseEndTime,
+      sampleStartTime: datesEnv.sampleStartTime,
+      sampleEndTime: datesEnv.sampleEndTime,
+    }
+    setDates(tmpDates)
+  }, [])
```

Comment:
You should have datesEnv in the list there `}, [datesEnv])`

Reply:
if I do that, I get in the js console `Error: Objects are not valid as a React child [...]. If you meant to render a collection of children, use an array instead.`

i don't really understand that yet, but the whole point of this is to keep these from updating when the context changes so... maybe leaving it out is fine? i modeled it after the example two above

Reply:
If it works then that's fine, usually useEffect needs to know what states it's dependent on to execute, otherwise it should only ever run once